src/Bridge/SetRectorsResolver.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Bridge;
+
+use Rector\Config\RectorConfig;
+use Rector\Contract\Rector\RectorInterface;
+use ReflectionProperty;
+use Webmozart\Assert\Assert;
+
+/**
+ * @api
+ * @experimental since 1.1.2
+ * Utils class to ease building bridges by 3rd-party tools
+ */
+final class SetRectorsResolver
+{
+    /**
+     * @return array<class-string<RectorInterface>>
+     */
+    public function resolveFromFilePath(string $configFilePath): array
+    {
+        Assert::fileExists($configFilePath);
+
+        $rectorConfig = new RectorConfig();
+
+        /** @var callable $configCallable */
+        $configCallable = require $configFilePath;
+        $configCallable($rectorConfig);
+
+        // get tagged class-names
+        $tagsReflectionProperty = new ReflectionProperty($rectorConfig, 'tags');
+        $tags = $tagsReflectionProperty->getValue($rectorConfig);
+
+        $rectorClasses = $tags[RectorInterface::class] ?? [];
+        sort($rectorClasses);
```

Comment:
is `sort()` should only happen on reporting? for levelling rule, the original queue seems matter

Reply:
It's just to make output alphabetical and easy to scan through by eyes :)